frontends/mit-open/src/pages/HomePage/HomePage.tsx

```diff
@@ -81,16 +30,9 @@ const HomePage: React.FC = () => {
           <HeroSearch />
         </Container>
       </FullWidthBackground>
-      <Container>
-        <section>
-          <h2>Upcoming Courses</h2>
-          <TabbedCarousel config={UPCOMING_COURSES_CAROUSEL} />
-        </section>
-        <section>
-          <h2>Media</h2>
-          <TabbedCarousel config={MEDIA_CAROUSEL} />
-        </section>
-      </Container>
+      <UpcomingCoursesSection />
+      <MediaSection />
+      <BrowseTopicsSection />
```

Comment:
I was going to say
```ts
<Container>
      <UpcomingCoursesSection />
      <MediaSection />
      <BrowseTopicsSection />
</Container>
```
might be good, but I guess we need each section to have its own container in order to support the full-width backgrounds?